addons/mail/static/src/messaging/component/follower_list_menu/follower_list_menu_tests.js

```diff
@@ -124,6 +125,172 @@ QUnit.test('base rendering editable', async function (assert) {
     );
 });
 
+QUnit.test('base rendering with multiple partner followers', async function (assert) {
+    assert.expect(7);
+
+    await this.start({
+        async mockRPC(route, args) {
+            if (
+                route === '/web/dataset/call_kw/res.partner/read' &&
+                args.args[1][0] === 'message_follower_ids'
+            ) {
+                return [{
+                    message_follower_ids: [1, 2],
+                }];
+            }
+            return this._super(...arguments);
+        },
+    });
+    this.data['mail.followers'].records = [
+        {
+            // simulate real return from RPC
+            // (the presence of the key and the falsy value need to be handled correctly)
+            channel_id: false,
+            id: 1,
+            name: "Mitchell Admin",
+            partner_id: this.env.session.partner_id,
+        }, {
+            // simulate real return from RPC
+            // (the presence of the key and the falsy value need to be handled correctly)
+            channel_id: false,
+            id: 2,
+            name: "Eden Hazard",
+            partner_id: 11,
+        },
+    ];
+    const thread = this.env.entities.Thread.create({
+        id: 100,
+        model: 'res.partner',
+    });
+    thread.refreshFollowers();
```

Comment:
that sucks... I think it's best to pass `followers` as param of `Thread.create()`

Reply:
The idea is to simulate the rpc call which was causing the bug at the beginning (the problem was in `convertData`) so passing `followers` to `Thread.create()` would not test the code I want to test ..

Reply:
this line is problematic, because test is too dependent on implementation detail. The follower_list_menu present data from thread, but the fact it comes from simulated `.refreshFollowers` should not be apparent.

Maybe should have test in chatter, because this comes from chatter flow?